sdk/storage/azure-storage-blob/azure/storage/blob/_models.py

```diff
@@ -552,16 +508,16 @@ def __init__(self, **kwargs):
         self.encryption_scope = kwargs.get('x-ms-encryption-scope')
         self.request_server_encrypted = kwargs.get('x-ms-server-encrypted')
 
-        # these are for delete_blobs settings
-        self.delete_snapshots = kwargs.pop('delete_snapshots', None)
-        self.lease_id = kwargs.pop('lease_id', None)
-        self.if_modified_since = kwargs.pop('if_modified_since', None)
-        self.if_unmodified_since = kwargs.pop('if_unmodified_since', None)
-        self.match_condition = kwargs.pop('match_condition', None)
-        self.timeout = kwargs.pop('timeout', None)
-
-        # these are for set_standard_blob_tier_blobs settings
-        self.rehydrate_priority = kwargs.pop('rehydrate_priority', None)
+        # # these are for delete_blobs settings
```

Comment:
is it okay to remove these commented lines of code?

Reply:
good catch, forgot to delete